client/components/message-user/message-user.jsx

```diff
@@ -0,0 +1,80 @@
+import { Meteor } from 'meteor/meteor';
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Modal, Button, Form, Message } from 'semantic-ui-react';
+
+class MessageUserModal extends Component {
+    constructor(props) {
+	super(props);
+	this.state = { text: '', user: props.user, error: null };
+    }
+
+    render() {
+	const user = this.state.user;
+	const open = Boolean(this.state.user);
+	if (!open) {
+	    return null;
+	}
+
+	const { text } = this.state;
+
+	return (
+	    <Modal
+		size="large"
+		open={true}
+		closeIcon={true}
+		onClose={() => this._clearState() }
+	    >
+		<Modal.Header>Message {user.name}</Modal.Header>
+		<Modal.Content>
+		    <Form onSubmit={(e) => this._sendMessage(e)}>
+			<Form.TextArea label='Message:' placeholder='Text...' name='text' value={text} onChange={this._handleChange} />
+			<Form.Group>
+			    <Form.Button color='green' type='submit' content='Send' />
+			    <Form.Button floated='right' color='red' inverted type='submit' content='Cancel' onclick={() => this._clearState() } />
+			</Form.Group>
+			<p> This will send an email message to the player. The email message will include your message and email address for them to reply to you.</p>
+			<Message negative
+				 hidden={!this.state.error}
+				 icon='warning sign'
+				 content={this.state.error ? this.state.error : ''}
+				 />
+		    </Form>
+		</Modal.Content>
+	    </Modal>
+	);
+    }
+
+    _handleChange = (e, { name, value }) => this.setState({ [name]: value })
+
+    _sendMessage(e) {
```

Comment:
Would it be possible to change `user.message` to take a user_id instead of an email address? I'd rather keep a user's email address out of the local browser memory if possible. (That's probably something that should not be published anyways - I'll make a separate card about it.) The server method can do the lookup for the team captain's email address.

Reply:
Can be done, but this will be a more extensive change because other email functions also take an address rather than a user id. This should be a separate issue.

Reply:
Thanks for the info there. I have a Trello card to try to remove email from the published data so I'll probably try to address this comment if I ever get to that card